rosbag2_tests/test/rosbag2_tests/test_rosbag2_record_end_to_end.cpp

```diff
@@ -55,6 +55,8 @@ std::shared_ptr<test_msgs::msg::Strings> create_string_message(
 }  // namespace
 
 TEST_F(RecordFixture, record_end_to_end_test) {
+  const auto bag_file_name = get_test_name();
+  set_path(bag_file_name);
```

Comment:
Could we do this in `SetUp`?

Reply:
I actually don't know. I'll give it a  try and report back...